aten/src/ATen/detail/MPSHooksInterface.h

```diff
@@ -90,6 +90,12 @@ struct TORCH_API MPSHooksInterface : AcceleratorHooksInterface {
   bool hasPrimaryContext(DeviceIndex device_index) const override {
     FAIL_MPSHOOKS_FUNC(__func__);
   }
+  virtual bool isPinnedPtr(const void* data) const override {
+    return false;
+  }
```

Comment:
@wizzniu and @albanD 
What is needed to adopt the MPS to the accelerator interface ? If there are APIs missing, we would like to extend it.

Reply:
@kulinseth Seems that there is no need to add extra APIs for MPS currently. But I don't test mps locally. It still need ci test for mps to verify.